Forge Networking Remastered Unity/Assets/Bearded Man Studios Inc/Scripts/Networking/Forge/Networking/FacepunchNetworkingPlayer.cs

```diff
@@ -0,0 +1,17 @@
+﻿#if FACEPUNCH_STEAMWORKS
+using Steamworks;
+
+namespace BeardedManStudios.Forge.Networking
+{
+    public class FacepunchNetworkingPlayer : NetworkingPlayer
```

Comment:
Is this needed? Could we not use the existing `SteamNetworkingPlayer` instead?

Reply:
The SteamNetworkingPlayer uses CSteamId type, whereas Facepunch.Steamworks uses SteamId. It might be the case that this could be removed entirely and just use the NetworkingPlayer constructor for the FACEPUNCH_STEAMWORKS case, but I haven't tested whether this is possible.